src/entities/BlockNode/BlockNode.spec.ts

```diff
@@ -0,0 +1,65 @@
+import { createBlockNodeMock } from '../../utils/mocks/createBlockNodeMock';
+import { createBlockTuneMock } from '../../utils/mocks/createBlockTuneMock';
```

Comment:
Those methods don't create mocks, they create real classes, so this test is actually an integration test.

For unit test, you should create mocked values and check that `serialized` getter is accessed for each mock